lib/exercism-config.rb

```diff
@@ -1,5 +1,6 @@
 require 'aws-sdk-dynamodb'
 require 'aws-sdk-secretsmanager'
+require 'octokit'
```

Comment:
This needs removing from here. This would mean every bit of the infrastructure needs the octokit gem installed. But because we don't specify it in our gemspec we can't guarantee that. So the whole thing will break. Instead we just require it on demand below.